### ISSUE TITLE
conjure-up breaks bsdtar (& possibly others) due to lib install hook

### ISSUE THREAD

**Issue by josvazg** [conjure-up/conjure-up#770]
conjure-up installs /etc/ld.so.conf.d/conjure-up.conf which points LD to:
/snap/conjure-up/140/usr/lib/x86_64-linux-gnu/

This breaks tools like bsdtar that use different versions of libraries held there, like for instance libachive.so.13 in case of bsdtar.

One would expect a snap not to break the environment and tools outside of it.

How to reproduce:
1. Install bsdtar with:
  $ sudo apt install bsdtar
2. Test it works:
  $ bsdtar
  bsdtar: Must specify one of -c, -r, -t, -u, -x

3. Install conjure-up snap

4. Test bsdtar fails:
  $ bsdtar
  bsdtar: symbol lookup error: bsdtar: undefined symbol: archive_read_add_passphrase

5. Test that ld path to libarchive.so.13 has now 2 entries:
  $ sudo ldconfig -p |grep libarchive
   	libarchive.so.13 (libc6,x86-64) => /snap/conjure-up/140/usr/lib/x86_64-linux-gnu/libarchive.so.13
	libarchive.so.13 (libc6,x86-64) => /usr/lib/x86_64-linux-gnu/libarchive.so.13

This breaks things like vagrant, that depend on bsdtar from libarchive-tools

**Comment by battlemidget** [comment]
What's the output of 

`snap list` and `snap info conjure-up`

**Comment by battlemidget** [comment]
I know I fixed this in a later revision, can you try:

```
sudo snap refresh conjure-up --beta
```

And let me know if any `snap` directories still end up in `ldconfig` output?

**Comment by josvazg** [comment]
```
$ snap list and snap info conjure-up
Name        Version  Rev  Developer  Notes
conjure-up  2.1.2    140  canonical  classic
```

**Comment by josvazg** [comment]
Yes, beta works fine:

```
$ sudo snap install conjure-up --beta --classic
conjure-up (beta) 2.2-beta1 from 'canonical' installed
$ snap list and snap info conjure-up
Name        Version    Rev  Developer  Notes
conjure-up  2.2-beta1  151  canonical  classic
$ sudo ldconfig 
$ sudo ldconfig -p |grep libarchive
	libarchive.so.13 (libc6,x86-64) => /usr/lib/x86_64-linux-gnu/libarchive.so.13
$ bsdtar
bsdtar: Must specify one of -c, -r, -t, -u, -x

```

**Comment by battlemidget** [comment]
Thanks I'll get a fix out for stable soon

**Comment by battlemidget** [comment]
This is fixed in the next 2.2 release
